### ISSUE TITLE
Corrupted Loot in Dungeon 

### ISSUE THREAD

**Issue by StayStreet21** [scalda/FoolCraft#831]
## SUGGESTION/HELP
- [ ] I would like to propose an additional mod, config change, custom recipe, etc.
- [ ] I am requesting help with installing or running the pack

## Issue Description
I was looting one of the 5 layered caves, and as i got to the last main room (with obsidian walls and like 8 spawners in the ceiling) in the chest was a "no name" item without a texture. I was able to pick it up and (stupidly) put it in my hot bar, I scrolled over the item and the game crashed. When I scrolled over the item it said it was a ghast tear chicken. I'm unsure if it was a spawn egg. My best guest is maybe when making the loot tables, the chicken itself is what was codded into it and not the actual spawn egg. Unless the actual chicken egg 

## Reproduce Steps
Holding the item. Having it in my inventory didn't seem to do anything but when i scrolled over it, my game crashed and every time I load back in, I crash. The server owner is currently clearing it from my inventory using NBTEditor but so far we haven't had any luck getting me back in :/ 

## Additional Information

Crash report: https://pastebin.com/AcCCF1sz

**Comment by HamzaNa1** [comment]
i dont think that's a modpack problem it might be because of lagg the chest didnt load right, the only way i can think of to fix it is to delete your stuff using NBT and then use creative to get them back

**Comment by HamzaNa1** [comment]
or it might corrupted the chunk that why u cant join maybe try to change your position using NBT too

**Comment by StayStreet21** [comment]
As soon as it happened I was able to remove the item with NBT and that took care of it. I reported it to here just incase it was something to do with the modpack and I didn't want other people who might not know enough to run into the same problem. I didn't know that lag could cause a chest to spawn items in like that. I thought maybe that when making the loot tables for those dungeons, the "item" was actually the ghost tear chicken entity itself because the item didn't say spawn egg like the rest. All of the other things in that chest were fine and I didn't have a problem with any of the other spawn eggs I received from them. It was weird, but it's fixed on my end. I just didn't want other people to run into this issue and not know how to fix it. 

Thank you again!
